sites/cheerpj/src/content/docs/13-tutorials/05-interoperability-tutorial.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Java and JavaScript Interoperability
+description: Learn how to enable Java and JavaScript communication using CheerpJ.
+---
+
+With CheerpJ, a Java application can integrate with JavaScript in a browser environment, enabling seamless interaction between Java and JavaScript code.
+
+In this tutorial, we’ll explore creating a two-way communication channel between Java and JavaScript using CheerpJ. The idea is to allow a Java class to access JavaScript functions and vice versa, to enable sending data between the two environments.
```

Comment:
Maybe we can say ` we'll teach you how to create a two-way communication channel` instead of the `'we'll explore`